### ISSUE TITLE
Ensemble models

### ISSUE THREAD

**Issue by enavarrocomes** [maxpumperla/hyperas#77]
Hey,
When using the best_ensemble method (for 5 models), and analyzing each one of those models returned, I find that the best one over the validation set is not the first one (in order). What models are exactly returned? Shouldn't it be the best n models over the validation set?

Thank you

**Comment by maxpumperla** [comment]
The ensemble is created from whatever loss you let hyperopt optimise for, e.g. in this example by the following:

https://github.com/maxpumperla/hyperas/blob/master/examples/mnist_ensemble.py#L50-L52

if you want val loss, use that instead
